drivers/rtc/qm_rtc.c

```diff
@@ -117,7 +117,7 @@ int qm_rtc_read(const qm_rtc_t rtc, uint32_t *const value)
 {
 	QM_CHECK(rtc < QM_RTC_NUM, -EINVAL);
 	QM_CHECK(value != NULL, -EINVAL);
-	*value=QM_RTC[rtc].rtc_ccvr;
+	*value = QM_RTC[rtc].rtc_ccvr;
```

Comment:
You can rebase your branch on top of master locally, then do a push -f to your remote branch. This will update this Pull Request. Let me know that works, ok?